### PR TITLE
Fix the check for an available update

### PR DESCRIPTION
**Pull request by notmgsk**

### CODE REVIEW

File: app/src/entry-point.lisp [hunk 1]
```diff
@@ -268,11 +268,12 @@
     (check-libraries))
   #+forest-sdk
   (unless skip-version-check
+    (format t "hi")
```
Comment (ecpeterson) [inline app/src/entry-point.lisp]: get rid of this please

Comment (notmgsk) [reply app/src/entry-point.lisp]: 😨

**Comment by stylewarning** [comment]
can we be robust to not having a network connection?

File: app/src/versions.lisp [hunk 2]
```diff
@@ -38,13 +38,17 @@
...
 (defun latest-sdk-version ()
   "Get the latest SDK quilc version, or NIL if unavailable."
-  (let* ((s (drakma:http-request "http://downloads.rigetti.com/qcs-sdk/version"
-                                 :want-stream t))
-         (p (yason:parse s)))
-    (multiple-value-bind (version success)
-        (gethash "quilc" p)
-      (when success
-        version))))
+  (handler-case
```
Comment (notmgsk) [inline app/src/versions.lisp]: @stylewarning like this?

**Pull request merged**